docs/integration/OpenAPI.md

```diff
@@ -66,8 +66,8 @@ components {
 
 | Parameter      | Required | Default | Description                                                                                                                   |
 |----------------|----------|---------|-------------------------------------------------------------------------------------------------------------------------------|
-| url            | true     |         | URL with OpenAPI resource                                                                                                     |
```

Comment:
To me it was confusing what we (OpenAPI specification?) mean by 'resource'.  The definition? The service itself?
So, what we meant here is called "Open API  document".

Reply:
I'd say "XXX HTTP service documentation in the OpenAPI format" when I wanted to talk about the resource which points to the doc. 

So you can have an HTTP service that can be described without OpenAPI but which follows the OpenAPI rules. 
When you add the OpenAPI description of the endpoints defined in the service, you will have human-readable documentation and machine-readable documentation at the same time.

The Enricher is the machine I mentioned. It reads the documentation of the external service and thanks to the well-defined and known structure of the documentation (the Open API format) it can interact with the service without additional (e.g. programmatical) effort. 

There are various ways we could pass the documentation to the Enricher. In the current implementation, it's an HTTP link.  But it could be also a file uploaded. It doesn't matter. 

On the other hand in the OpenAPI yaml file, there should be a "servers" field (https://spec.openapis.org/oas/latest.html#serverObject) where you (as an author of the HTTP service) can define the URL of your service (you tell your users: "hey, my XXX HTTP service is here"). 

Both fields are important for the Enricher. The first one is needed to "understand" how to interact with the service. The second one, when we know how to interact with the service, is used to do the interaction (because the Enricher knows now where is the location of the service).

----

It was the theory. 
So in our case: 
1. The "URL" is a link to the documentation of the HTTP (aka REST) service defined in the Open API format
2. The "rootUrl" is sth that instead of the "servers" section from the Open API (at least I understand it like that). 

So, both can be improved. WDYT?